source/_integrations/media_extractor.markdown

```diff
@@ -76,3 +76,12 @@ This will download the file from the given URL.
 | `entity_id`            | yes      | Name(s) of entities to seek media on, e.g., `media_player.living_room_chromecast`. Defaults to all.       |
 | `media_content_id`     | no       | The ID of the content to play. Platform dependent.                                                        |
 | `media_content_type`   | no       | The type of the content to play. Must be one of MUSIC, TVSHOW, VIDEO, EPISODE, CHANNEL or PLAYLIST MUSIC. |
+
+### Cookies
+
+Some supported video sites are enhanced by a logged in user.
```

Comment:
```suggestion
Some supported video services provide a better experience when you are logged in. For example, if you use Twitch Turbo or are subscribed to a Twitch streamer, logging in prevents the video stream from showing an "An ad is currently playing. Come back after the break." message until the ads are over. Other examples are private or purchased YouTube videos which you can only watch while logged in.
```
I like the explanation you provided in the core PR. I think it would be helpful to have it in the docs, as well. So I took the liberty of copying it over here.